aspnetcore/tutorials/web-api-help-pages-using-swagger/samples/2.1/TodoApi.NSwag/Startup.cs

```diff
@@ -28,10 +28,13 @@ public void Configure(IApplicationBuilder app)
 
             // Register the Swagger generator and the Swagger UI middlewares
             app.UseOpenApi();
-            app.UseSwaggerUi3();
+            if (app.Environment.IsDevelopment())
+            {
+                app.UseSwaggerUi3();
+            }
```

Comment:
This doesn't compile, apparently because `app` here is `IApplicationBuilder` and doesn't have `Environment`. In later versions `app` is `WebApplication'.

Reply:
> This doesn't compile, apparently because `app` here is `IApplicationBuilder` and doesn't have `Environment`. In later versions `app` is `WebApplication'.

2.1 is out of support so we're only interested in the intent.

Reply:
Not technically related to this task but do we keep samples around for out-of-support versions?

Reply:
> Not technically related to this task but do we keep samples around for out-of-support versions?

We've been keeping out of support content.
We should probably at least add a comment to call `UseSwaggerUi3` on in development.